src/gui/calibration.rs

```diff
@@ -0,0 +1,166 @@
+use crate::config::{GainPresets, Linearize};
+use crate::spectrum;
+use egui::{Button, ComboBox, Context, Slider};
+
+pub struct CalibrationWindow {}
```

Comment:
Some of the windows have no state, and look like this. For consistency I made them structs anyway. If you don't like this style we can throw away the structs and instead have just a bare `update` function. That would work, but the different modules would not follow the same structure then.

Reply:
This is a good approach. Makes extending in the future easier and I do not see any obvious downsides.